java/core/src/main/java/com/microsoft/bond/BondedBondType.java

```diff
@@ -79,30 +81,35 @@ public final boolean isGenericType() {
 
     @Override
     protected final Bonded<TStruct> newDefaultValue() {
-        return new Bonded<TStruct>(this.valueType.newDefaultValue());
+        // since Bonded is immutable, the default value can be shared
```

Comment:
Bonded is immutable: the underlying stream/object is cloned but never modified directly. The default value wraps a default object value. 

---
In reply to: [128378278](https://github.com/Microsoft/bond/pull/532#discussion_r128378278) [](ancestors = 128378278)